master/test/streamdata/stream_trials.sql

```diff
@@ -1,8 +1,15 @@
 -- Trials to insert for test
+INSERT INTO workspaces (name) VALUES ('test_workspace1');
```

Comment:
nit (non-blocking): would it make sense to also change this name to be consistent with the function?